iamidentityv1/iam_identity_v1.go

```diff
@@ -43,7 +43,7 @@ type IamIdentityV1 struct {
 }
 
 // DefaultServiceURL is the default URL to make service requests to.
-const DefaultServiceURL = "https://iam.cloud.ibm.com"
+const DefaultServiceURL = "https://iam.test.cloud.ibm.com"
```

Comment:
You generated the code using the staging version of the API definition again, didn't you? :joy:
You'll need to re-gen the code using the production version of the API definition

Reply:
Also, please upgrade to the very latest version of the SDK generator before you re-gen the code using the production API defn.